dfm/train.py

```diff
@@ -0,0 +1,182 @@
+"""Training script for training masked language models with sweeps
+
+Authors:
+    Lasse Hansen
+    Malte Højmark-Bertelsen
+
+```bash 
+PYTHONPATH="." python dfm/train.py --path_to_config_file
+```
+"""
+
+from dataclasses import dataclass
+from typing import List, Optional
+
+from transformers import (
+    AutoConfig,
+    AutoModelForPreTraining,
+    Trainer,
+    TrainingArguments,
+    AutoTokenizer,
+    DataCollatorForLanguageModeling,
+)
+from datasets import interleave_datasets
+
+from dfm.data.load import dfm_load_dataset
+from dfm.modelling.preprocess import preprocess_dataset
+from dfm.modelling.utils import read_yaml
+from dfm.modelling.model_types import MODEL_TYPES
+
+
+def main():
```

Comment:
Missing docstring

Reply:
Still missing docstring? Why was this resolved?

Reply:
Fixed now.